frontends/web/src/components/devices/bitbox02/bitbox02.tsx

```diff
@@ -113,6 +114,15 @@ class BitBox02 extends Component<Props, {}> {
                     <Dialog onClose={() => this.channelVerify(false)}>
                         <p>Verify your BitBox</p>
                         <p>{hash}</p>
+
+                        { !deviceVerified ?
+                            <p>First, verify the hash on your device</p>
+                        :
+                        <div className={'flex flex-1 flex-row flex-items-center spaced'}>
+                            <p>Device Hash: {hash}</p>
```

Comment:
Unsure what this does. The point was that the hash is displayed on both sides, and after it's confirmed on the device, the hash is not shown anymore on the screen (because it is not visible anymore on the device).

Here the hash is shown twice after confirm on the device?

After device confirmation, the app should ask the user if they confirmed or rejected on the device, so the app confirmation works without showing the hash after the device confirmation/rejection.